fluidreview/api.py

```diff
@@ -212,7 +212,9 @@ def parse_webhook(webhook):
         webhook.status = WebhookParseStatus.SUCCEEDED
     except:  # pylint: disable=bare-except
         webhook.status = WebhookParseStatus.FAILED
-        log.exception('Webhook %s body is not valid JSON or has invalid/missing values.', webhook.id)
+        message = 'Webhook %s is missing values for award_cost and/or amount_to_pay', webhook.id
+        log.exception(message)
+        raise Exception(message)
```

Comment:
Remove this line and update `bootcamp.settings.LOGGING` with the following ('sentry' added to `django` and `root` handlers):

```python
    "loggers": {
        "django": {
            "propagate": True,
            "level": DJANGO_LOG_LEVEL,
            "handlers": ["console", "syslog", "sentry"],  
        },
        "django.request": {
            "handlers": ["mail_admins"],
            "level": DJANGO_LOG_LEVEL,
            "propagate": True,
        },
        "elasticsearch": {"level": ES_LOG_LEVEL},
        "raven": {"level": SENTRY_LOG_LEVEL, "handlers": []},
        "nplusone": {"handlers": ["console"], "level": "ERROR"},
    },
    "root": {"handlers": ["console", "syslog", "sentry"], "level": LOG_LEVEL},
```

Reply:
https://sentry.io/mit-office-of-digital-learning/bootcamp/issues/856760643/

Reply:
@mbertrand `"elasticsearch": {"level": ES_LOG_LEVEL},` this line does not exists in current code, do you want to add this and remove `'urllib3': {'level': 'INFO',}`